src/renderer/store/modules/utils.js

```diff
@@ -553,6 +555,17 @@ const actions = {
           hashtag
         }
       }
+
+      case 'post': {
+        const match = url.pathname.match(postPattern)
+        const postId = match.groups.postId
+        const query = { ucid: url.searchParams.get('ucid') }
```

Comment:
Would it be possible we use a more self-explanatory parameter here (e.g. `authorId`), as I didn't realise what `ucid` was until I read the rest of the code in this pull request?